src/Network/Send.pm

```diff
@@ -407,6 +407,17 @@ sub reconstruct_game_login {
 	my ($self, $args) = @_;
 	$args->{userLevel} = 0 unless exists $args->{userLevel};
 	($args->{iAccountSID}) = $masterServer->{ip} =~ /\d+\.\d+\.\d+\.(\d+)/ unless exists $args->{iAccountSID};
+
+	if (exists $args->{mac}) {
+		my $key = pack('C16', (0x06, 0xA9, 0x21, 0x40, 0x36, 0xB8, 0xA1, 0x5B, 0x51, 0x2E, 0x03, 0xD5, 0x34, 0x12, 0x00, 0x06));
+		my $chain = pack('C16', (0x3D, 0xAF, 0xBA, 0x42, 0x9D, 0x9E, 0xB4, 0x30, 0xB4, 0x22, 0xDA, 0x80, 0x2C, 0x9F, 0xAC, 0x41));
+		my $mac = $config{mac} || "F2ADCC03771E";
```

Comment:
I propose to move `mac` parameter to **server.txt**

Reply:
this was made to spoof the user MAC, if move to servers.txt user will have to clone the entire openkore folder to change.
here in brazil and in thailand some users use proxy so make sense use 1 mac per 1 proxy

Reply:
I think use `$config{mac_address}` better ?

Reply:
> I think use `$config{mac_address}` better ?

+1